client/src/pages/Dashboard/dashboardStyles.ts

```diff
@@ -10,6 +10,7 @@ const useStyles = makeStyles((theme) => ({
   },
   board: {
     marginTop: 30,
+    minHeight: '40vh',
```

Comment:
With the minimum height on my laptop the board is out of place and too far down the screen than before, is this for the SplashScreen component?

Reply:
Oops, yes. This should not be there.